src/HTTP/Wrapper/SuperGlobalDropInReplacement.php

```diff
@@ -54,6 +54,13 @@ public function offsetSet($offset, $value): void
      */
     public function offsetUnset($offset): void
     {
-        throw new LogicException("Modifying global Request-Array such as \$_GET is not allowed!");
+        if ($offset === 'ticket') {
+            // Allow phpCAS to unset the 'ticket'
+            // they do this in CAS/Client.php#L1070
+            // unset($_GET['ticket']);
+            parent::offsetUnset($offset);
+        } else {
+            throw new LogicException("Modifying global Request-Array such as \$_GET is not allowed!");
+        }
```

Comment:
Please revert this change and follow the instructions here: https://github.com/ILIAS-eLearning/ILIAS/tree/release_8/src/HTTP#dropinreplacements